components/blitz/src/omero/client.java

```diff
@@ -329,7 +329,10 @@ private void init(Ice.InitializationData id) {
 
         // Strictly necessary for this class to work
         optionallySetProperty(id, "Ice.ImplicitContext", "Shared");
-        if (Ice.Util.intVersion() <= 30500) {
+        if (Ice.Util.intVersion() >= 30600) {
+            optionallySetProperty(id, "Ice.ACM.Client.Timeout", "60000");
```

Comment:
It might be worth putting these into `Constants.ice`. If these were more server-side properties I'd suggest also duplicating in [omero.properties](https://github.com/openmicroscopy/openmicroscopy/blob/v5.2.2/etc/omero.properties#L878) so that they get generated to https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/config.html -- we don't currently have that kind of page for users. Perhaps a docs RFE for later would be to generate a user-friendly page from https://github.com/openmicroscopy/openmicroscopy/blob/v5.2.2/etc/ice.config as we do for omero.properties.